modules/islandora_text_extraction/README.md

```diff
@@ -1,44 +1,49 @@
-# islandora_text_extraction
-### Connects Islandora 8 to Hypercube microservice and extracts text from PDFs
+# Islandora Text Extraction `
```

Comment:
Back-tick shouldn't be here.

Reply:
I've got more submodules to go.... I'll take care of it

Reply:
Ok. I'll just hold off until I hear from you that it is ready to go.